apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerDbTypeTests.ValueConversions.cs

```diff
@@ -147,130 +162,149 @@ public static IEnumerable<object[]> GetValidValueConversions()
+            yield return new object[] { 11, SpannerDbType.Bool, "true", TestType.ClrToValue };
+            yield return new object[] { 0, SpannerDbType.Bool, "false" };
+            yield return new object[] { 1U, SpannerDbType.Bool, "true" };
+            yield return new object[] { 100U, SpannerDbType.Bool, "true", TestType.ClrToValue };
+            yield return new object[] { 0U, SpannerDbType.Bool, "false" };
+            yield return new object[] { 1L, SpannerDbType.Bool, "true" };
+            yield return new object[] { -1L, SpannerDbType.Bool, "true", TestType.ClrToValue };
+            yield return new object[] { 0L, SpannerDbType.Bool, "false" };
+            yield return new object[] { (ulong)1, SpannerDbType.Bool, "true" };
+            yield return new object[] { (ulong)21, SpannerDbType.Bool, "true", TestType.ClrToValue };
+            yield return new object[] { (ulong)0, SpannerDbType.Bool, "false" };
+            yield return new object[] { (short)1, SpannerDbType.Bool, "true" };
+            yield return new object[] { (short)-1, SpannerDbType.Bool, "true", TestType.ClrToValue };
+            yield return new object[] { (short)0, SpannerDbType.Bool, "false" };
+            yield return new object[] { (ushort)1, SpannerDbType.Bool, "true" };
+            yield return new object[] { (ushort)11, SpannerDbType.Bool, "true", TestType.ClrToValue };
+            yield return new object[] { (ushort)0, SpannerDbType.Bool, "false" };
 
             // Spanner type = String tests.
             // Note the casing on bool->string follows c# bool conversion semantics (by design).
-            yield return new object[] {true, SpannerDbType.String, Quote("True")};
-            yield return new object[] {false, SpannerDbType.String, Quote("False")};
-            yield return new object[] {(char) 26, SpannerDbType.String, Quote("\\u001a")};
-            yield return new object[] {(byte) 1, SpannerDbType.String, Quote("1")};
-            yield return new object[] {(sbyte) 1, SpannerDbType.String, Quote("1")};
-            yield return new object[] {1.5M, SpannerDbType.String, Quote("1.5")};
-            yield return new object[] {1.5D, SpannerDbType.String, Quote("1.5")};
-            yield return new object[] {1.5F, SpannerDbType.String, Quote("1.5")};
-            yield return new object[] {1, SpannerDbType.String, Quote("1")};
-            yield return new object[] {1U, SpannerDbType.String, Quote("1")};
-            yield return new object[] {1L, SpannerDbType.String, Quote("1")};
-            yield return new object[] {(ulong) 1, SpannerDbType.String, Quote("1")};
-            yield return new object[] {(short) 1, SpannerDbType.String, Quote("1")};
-            yield return new object[] {(ushort) 1, SpannerDbType.String, Quote("1")};
-            yield return new object[] {s_testDate, SpannerDbType.String, Quote("2017-01-31T03:15:30.5Z")};
+            yield return new object[] { true, SpannerDbType.String, Quote("True") };
```

Comment:
Most of these changes are white space only. We usually, specially if it is this much, don't include them on the same commit as actual changes, because it makes the actual changes harder to find a review.

Reply:
Sure Amanda, will keep this in mind going forward.